Merge upstream changes r1131 to r1145

ocean90 opened the pull request:
Fixes #11.

ocean90 committed:
Warnings: Make warnings for newline mismatches more verbose.

Previously a user had to check himself if a translation needs to begin/end on a newline or not because the warning wasn't specific enough. To help translators split `warning_both_begin_end_on_newlines()` into four warnings.

Props SergeyBiryukov.
Fixes #495.

markoheijnen committed:
Restyle the header which now is a top bar with title & navigation.
Breadcrumbs got a redesign and is below the top bar.
Action `after_hello ` did got removed and is replaced with a new filter `gp_nav_menu_items `

Fixes #255
Fixes #496

markoheijnen committed:
Update unit tests for updated breadcrumb function

See #496

markoheijnen committed:
Show login link on install.php again and redirect after login to the home page.

Fixes #460

ocean90 committed:
Locales: Add Emoji locale definition.

Fixes #499.

Commit:
Allow Glossaries to highlight multiple instances of a term in an original.
Fixes #498

ocean90 committed:
Glossary: Use the correct object type for translation sets in `gp_link_glossary_edit_get()`.

Fixes #501.

ocean90 committed:
Translation Import: Include fuzzy translations in the current translations list.

Translation files can contain fuzzy translations too, the translation has the flag "fuzzy". To avoid importing them again we should handle fuzzy translations as "current".

Fixes #470.

ocean90 committed:
Translation Matching: Respect permissions when propagating translations.

Set the translation status to "waiting" if the current user cannot approve translations in a translation set of a matching result.
Includes unit tests and moves the cookie handler to `GP_UnitTestCase` to avoid "headers already sent" errors.

See #327.

ocean90 committed:
Add Cebuano locale definition.

Fixes #503.

toolstack commented:
I tested this on my local install without any issues, the only item I saw when looking through the changes was in gp-includes/template-links.php, line 102.  It looks like we changed the permissions to let WP admins manage glossaries (probably as a temp hack) and this will revert that change (probably a good thing overall).

ocean90 commented:
> It looks like we changed the permissions to let WP admins manage glossaries (probably as a temp hack) and this will revert that change (probably a good thing overall).

Correct, the upstream fix was https://glotpress.trac.wordpress.org/changeset/1142. I have changed it back for consistency. There are 13 matches across 3 files for `->can( 'approve', 'translation-set'`.
